UnityProject/Assets/Scripts/Items/Implants/Organs/BodyFat.cs

```diff
@@ -18,9 +18,6 @@ public class BodyFat : BodyPartFunctionality, IMovementEffect
 
 		public float CrawlingSpeedModifier { get; private set; }
 
-
-		public Stomach RelatedStomach;
```

Comment:
why does this need to be removed to fix blob?

Reply:
> why does this need to be removed to fix blob?

It's null as it isnt set anywhere and isnt used so i just removed it

Reply:
Also causes NRE for blob spawn as it gibs, so this currently breaks all gibbing

Reply:
why do Blobs have BodyFat?

Reply:
it should be set when spawned via stomach

Reply:
> why do Blobs have BodyFat?

they dont, the player the gib out of does

Reply:
yeah  
var Added = Spawn.ServerPrefab(BodyFatToInstantiate.gameObject).GameObject.GetComponent<BodyFat>();
				BodyFats.Add(Added);
				
			var Added = Spawn.ServerPrefab(BodyFatToInstantiate.gameObject).GameObject.GetComponent<BodyFat>();
				Added.SetAbsorbedAmount(0);
needs to be set  in those two places in stomach

Reply:
done